swift/example_code/iam/README.md

```diff
@@ -0,0 +1,71 @@
+# Amazon IAM code examples for the SDK for Swift
+## Overview
+This folder contains code examples demonstrating how to use the AWS SDK for
+Swift to use the Amazon Identity and Access Management (`IAM`). This README
```

Comment:
the Amazon Identity and Access Management --> AWS Identity and Access Management

Reply:
Instead I went with changing it to "the Amazon Identity and Access Management service".

Reply:
It's "AWS" as opposed to "Amazon." First use in a file is "AWS Identity and Access Management (IAM)".